File: LeetCode/26. Remove Duplicates from Sorted Array/Solution.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        lastPos = 0
        for i in range(1, len(nums), 1):
            if nums[i] != nums[lastPos] :
                lastPos += 1
                nums[lastPos] = nums[i]
        return lastPos + 1
        

File: LeetCode/26. Remove Duplicates from Sorted Array/Solution.cs
public class Solution
{
    public int RemoveDuplicates(int[] nums)
    {
        var lastPos = 0;
        var len = nums.Length;

        for (var i = 1; i < len; i++)
            if (nums[i] != nums[lastPos]) nums[++lastPos] = nums[i];
        return lastPos + 1;
    }
}